packages/rollup-plugin/test/rollup-config-file-resolver.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { expect } from 'chai';
+import { rollupRunner } from './test-kit/rollup-runner';
+import { getProjectPath } from './test-kit/test-helpers';
+
+const getColor = () => getComputedStyle(document.body).color;
+
+describe('StylableRollupPlugin', function () {
+    this.timeout(20000);
+
+    const project = 'config-file-resolver-override';
+
+    const runner = rollupRunner({
+        projectPath: getProjectPath(project),
+        entry: './src/index.js',
+        pluginOptions: {
+            stcConfig: './project/stylable.config.js',
```

Comment:
did you have to provide this? is this because of how the tests are arranged?

Reply:
This is because of our setup, I believe for most project structures, passing "true" here will work as intended